README.md

```diff
@@ -9,6 +9,28 @@
 
 A flexible front end to plate bases pipelines in Sequencescape
 
+## Initial Setup
+
+1. In Sequencescape, configure all Limber required data:
+```shell
+  bundle exec rake limber:setup
+```
+
+1. Start Sequencescape (will start in 3000):
+```shell
+  bundle exec rails s
+```
+
+1. In Limber, connect to Sequencescape to configure required data:
+```shell
+  bundle exec rake config:generate
+```
+
+1. Start Limber (will start in 3001):
+```shell
+  bundle exec rails s
+```
+
```

Comment:
Are all the steps supposed to be numbered 1? I've seen Phill do something similar, so I'm assuming there's some kind of reasoning behind it.

Reply:
It has like an auto-increment, however there are times where it doesnt work well. Just in case, I'll change it to the real numbering.